cognite/client/_http_client.py

```diff
@@ -107,24 +107,28 @@ def __init__(
         self.refresh_auth_header = refresh_auth_header
         self.retry_tracker_factory = retry_tracker_factory  # needed for tests
 
-    def request(self, method: str, url: str, **kwargs: Any) -> requests.Response:
+    def request(self, method: str, url: str, accept: str, **kwargs: Any) -> requests.Response:
```

Comment:
Would be nice to get away from the **kwargs here and in `_do_request`. Would be slightly more verbose, but wouldn't need to deal with this untyped dict, which would make more approachable IMO